Clicker/SectionControllers/PollSectionController.swift

```diff
@@ -27,12 +44,23 @@ class PollSectionController: ListSectionController {
     }
     
     override func cellForItem(at index: Int) -> UICollectionViewCell {
-        let cell = collectionContext?.dequeueReusableCell(of: EmptyStateCell.self, for: self, at: index) as! EmptyStateCell
-        return cell
+        switch (userRole) {
+        case .admin:
+            let cell = collectionContext?.dequeueReusableCell(of: AskedCard.self, for: self, at: index) as! AskedCard
+            socket.addDelegate(cell)
+            cell.configureWith(socket: socket, delegate: askedCardDelegate, poll: poll)
+            return cell
+        default:
```

Comment:
should not be using default role. instead, if this is for the rest of the cases you can do `case .enum1, .enum2...`

Reply:
How should I handle the `Switch must be exhaustive` error that appears if I only have cases for the two members of the enum since `userRole` is an optional? Default case that fails? Or a `guard let` to ensure `userRole` isn't `nil`?